util/stopper_test.go

```diff
@@ -237,3 +236,52 @@ func TestStopperClosers(t *testing.T) {
 		t.Errorf("expected true & true; got %t & %t", tc1, tc2)
 	}
 }
+
+func TestStopperNumTasks(t *testing.T) {
+	s := NewStopper()
+
+	var tasks []PendingTask
+	for i := 0; i < 3; i++ {
+		tasks = append(tasks, s.StartTask())
+		if numTypes, numTasks := len(s.tasks), s.NumTasks(); numTypes != 1 || numTasks != i+1 {
+			t.Errorf("stopper should have %d running tasks, got %d / %+v", i+1, s.NumTasks(), s.tasks)
```

Comment:
Use the `numTypes` and `numTasks` in the format string, instead of computing the values again.